docs/setup-live-tv.md

```diff
@@ -0,0 +1,85 @@
+---
+id: setup-live-tv
+title: Setting up live TV
```

Comment:
this file seems to repeat some of the content from docs/server/index.md, maybe we should combine them into one or figure out which one should be first

Reply:
Yeah, there's a bunch of re-writing that needs to be done, this was mainly intended to move files into Docusaurus and give them a decent structure, I didn't touch the content at all :smiley:

Reply:
yeah, and once the site is live, i'll be able to give better feedback on the structuring